src/main/java/com/vacation_bot/domain/models/VacationRequestBody.java

```diff
@@ -0,0 +1,35 @@
+package com.vacation_bot.domain.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+/**
+ * Represents requested vacation.
+ */
+@Getter
+@Setter
+@EqualsAndHashCode
+public class VacationRequestBody {
+
+    /**
+     * The name of user who request vacation.
+     */
+    @JsonProperty(required = true)
+    private String userName;
```

Comment:
The fields are named different in the task description: 
```
{
    "user-name": "Alex",
    "start-date": 123134538,     // from epoch millis
    "stop-date": 123435433      // from epoch millis
}
```
 So you should specify the valid json names in the `@JsonProperty`. For example:
`@JsonProperty( "user-name" )`
Otherwise you user request will be rejected.